lib/index.ts

```diff
@@ -25,7 +30,7 @@ async function buildDepTreeFromPackagesConfig(
   }
 }
 
-async function buildDepTreeFromCsproj(
+async function buildDepTreeFromProjectFile(
```

Comment:
- I believe `getDependencyTreeFromCsproj` should be renamed as well.

- This is a breaking change, since we are changing API, so it should be committed properly.